netaddr.go

```diff
@@ -778,6 +780,46 @@ func (p IPPrefix) Contains(addr IP) bool {
 	return true
 }
 
+// Overlaps reports whether p and o overlap at all.
+//
+// If p and o are of different address families or either have a zero
+// IP, it reports false. Like the Contains method, a prefix with a
+// 4-in-6 IP is still treated as an IPv6 mask.
```

Comment:
4-in-6 is an IPv6 transitional mechanism, not the "embed v4 in v6" thing. I think you want to say "a prefix with an IPv6-mapped IPv4 IP is ..."